modules/bpi/bpi.push.inc

```diff
@@ -191,24 +193,29 @@ function bpi_convert_to_bpi($node, $category, $audience, $with_images = FALSE, $
   $bpi_content['related_materials'] = $materials_drupal;
   $bpi_content['editable'] = (int) $editable;
   $bpi_content['authorship'] = ($authorship) ? FALSE : TRUE;
-  $bpi_content['images'] = array();
+  $bpi_content['assets'] = array();
 
   if ($with_images) {
     $image_fields = bpi_fetch_image_fields($node->type);
 
     if (!empty($image_fields)) {
       foreach ($image_fields as $field_name) {
-        $field_value = field_view_field('node', $node, $field_name);
-
-        if (!empty($field_value['#items'][0]['uri'])) {
-          $file_url = file_create_url($field_value['#items'][0]['uri']);
-          // Image pseudo-check.
-          if (@getimagesize($file_url)) {
-            $bpi_content['images'][] = array(
-              'path' => $file_url,
-              'alt' => '',
-              'title' => '',
-            );
+        $image_type = bpi_get_image_type($field_name, $node);
+        if (!empty($image_type)) {
+          $field_value = field_view_field('node', $node, $field_name);
```

Comment:
Should this be [`field_get_items()`](https://api.drupal.org/api/drupal/modules%21field%21field.module/function/field_get_items/7) instead?